code/game/gamemodes/meteor/meteors.dm

```diff
@@ -220,9 +220,6 @@ GLOBAL_LIST_INIT(meteorsC, list(/obj/effect/meteor/dust)) //for space dust event
 		meteor_effect()
 		qdel(src)
 
-/obj/effect/meteor/ex_act()
```

Comment:
Instead of 10 snowflake ex_act handlings for every child of /effect/, we simply do it all on effect. 

Incredible.